deployment/ccip/changeset/internal/deploy_home_chain.go

```diff
@@ -411,10 +486,18 @@ func BuildOCR3ConfigForCCIPHome(
 				}
 				parsed = pk.Bytes()
 			}
+
 			transmittersBytes[i] = parsed
 		}
+
 		// validate ocr3 params correctness
-		_, err := ocr3confighelper.PublicConfigFromContractConfig(false, ocrtypes.ContractConfig{
+		// TODO: this is super hacky, should not have to do this.
```

Comment:
Can we specify the action item here within code? I assume it would be to update the validation within ocr3confighelper.PublicConfigFromContractConfig right?

Reply:
I'm not super clear on it yet, updating the ocr3 validation within libocr seems like a tough ask because its a common library shared by all OCR plugins, and they may rely on such validation.

We may have to duplicate the code somehow or reach out to research to validate our approach here, for now this "works" but I can create a ticket to follow up.

Reply:
Created a ticket here: https://smartcontract-it.atlassian.net/browse/CCIP-6062